packages/feedback-react/src/types.ts

```diff
@@ -0,0 +1,6 @@
+export type FeedbackOptions = 1 | 2 | 3 | 4 | 5;
+
+export type FeedbackType = {
+    feedbackValue: FeedbackOptions;
+    message?: string;
+};
```

Comment:
Liker heller ikke så godt å ha "type" med i typenavnet. Dette får meg til å tenke at jeg skal velge en type tilbakemelding. Kanskje rett og slett bare `Feedback`? Er jo det det er.. Ser også at du bruker `feedbackValue` inni her, som understøtter navneforslaget mitt over 😅

```suggestion
export type Feedback = {
    feedbackValue: FeedbackValue;
    message?: string;
};
```

Reply:
Litt enig, tenker `FeedbackPayload` er enda riktigere